rosdep/python.yaml

```
@@ -6768,6 +6768,59 @@ python3-gnupg:
   openembedded: [python3-gnupg@meta-python]
   opensuse: [python3-python-gnupg]
   ubuntu: [python3-gnupg]
+python3-google-auth:
+  debian:
+    '*': [python3-google-auth]
+    stretch:
+      pip:
+        packages: [google-auth]
+  fedora: [python3-google-auth]
+  opensuse: [python3-google-auth]
+  rhel:
+    '*': [python3-google-auth]
+    '7': null
+  ubuntu: [python3-google-auth]
+python3-google-auth-httplib2:
+  debian:
+    '*': [python3-google-auth-httplib2]
+    buster:
+      pip:
+        packages: [google-auth-httplib2]
+    stretch:
+      pip:
+        packages: [google-auth-httplib2]
+  fedora:
+    pip:
+      packages: [google-auth-httplib2]
+  opensuse: [python3-google-auth-httplib2]
+  ubuntu:
+    '*': [python3-google-auth-httplib2]
+    bionic:
+      pip:
+        packages: [google-auth-httplib2]
```

```suggestion
python3-google-auth-httplib2:
  debian:
    '*': [python3-google-auth-httplib2]
    buster: null
  opensuse: [python3-google-auth-httplib2]
  ubuntu:
    '*': [python3-google-auth-httplib2]
    bionic: null
```

Rather than mixing pip and system keys I would suggest not providing definitions for platforms that do not support it from system.

pip packages are not usable in build farm builds so adding the pip key here would not allow releasing a package which depends on this key.

Fedora is the biggest casualty as it had only a pip key but it's also not a required target platform.